test/tests/functional/pbs_qstat_formats.py

```diff
@@ -593,6 +607,7 @@ def test_qstat_json_valid_job_special_env(self):
             ' -f -F json ' + str(jid)
         ret = self.du.run_cmd(self.server.hostname, cmd=qstat_cmd_json)
         qstat_out = "\n".join(ret['out'])
+        self.logger.info(qstat_out)
```

Comment:
Do you still need this debug output?

Reply:
It's very helpful to see the json output especially if the test fails.

Reply:
Can you put it in the except block then?

Reply:
Done.